src/tests/ftest/fault_injection/pool.py

```diff
@@ -29,7 +29,7 @@ def setUp(self):
         self.object_class = self.params.get("object_class", "/run/*")
         self.number_servers = len(self.hostlist_servers) - 1
 
-    def look_missed_request(self, cmd_stderr, msg=b"MS request error"):
+    def look_missed_request(self, cmd_stderr, msg=b"err:"):
```

Comment:
Hmm, this doesn't seem very specific.   Could it match something that isn't a missed request?  A simple grep shows 41 instances of that string within quotes.

Reply:
Agreed, it would be good to identify the commit that requires this change.